pkg/features/_topics.json

```diff
@@ -0,0 +1,20 @@
+{
+  "active": [
+    "config",
+    "nonreg",
+    "hostlevel",
+    "resacct",
+    "cache",
+    "stall",
+    "rmsched",
+    "rtetols",
+    "overhead",
+    "wlplacement",
+    "unsched",
+    "nonrt",
+    "taint",
+    "nodelabel",
+    "byres",
+    "tmpol"
+  ]
```

Comment:
we need to sketch a schema for this document. Would you mind start a (VERY) simple design doc? should stay in the `doc/` folder of this repo.
At very minimum we need some forward-looking features like document versioning, documentation of the fields and a way to deprecate stuff

Reply:
right, will handle that